astropy/io/ascii/tests/test_write.py

```diff
@@ -339,7 +340,7 @@
          ),
 ]
 
-test_def_masked_fill_value = [
+test_defs_masked_fill_value = [
```

Comment:
For consistency with `test_defs` and `test_defs_no_data`.  The prefix `test_defs` denotes a collection, while the individual test definition is the singular `test_def`.

"Pluralize test_defs_masked_fill_value for consistency"